imager/urls.py

```diff
@@ -1,10 +1,60 @@
 from django.conf.urls import patterns, include, url
+from django.conf.urls.static import static
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
+from django.core.urlresolvers import reverse_lazy
+from imager import views as main_views
+from registration.backends.simple.views import RegistrationView
+from django.conf import settings
 
-urlpatterns = patterns('',
-    # Examples:
-    # url(r'^$', 'imager.views.home', name='home'),
-    # url(r'^blog/', include('blog.urls')),
 
-    url(r'^admin/', include(admin.site.urls)),
-)
+class MyRegistrationView(RegistrationView):
+    def get_success_url(self, request, user):
+        return '/profile/'
+
+
+urlpatterns = patterns(
+      '',
```

Comment:
This URLconf really needs to be better organized so everything is compartmentalized in one url file per app.